test/uk/gov/hmrc/cdsimportsddsfrontend/controllers/DeclarationController_goodsIdentificationSpec.scala

```diff
@@ -66,7 +73,15 @@ with Scenarios with FutureAwaits with BeforeAndAfterEach {
         "goodsIdentification.description" -> Seq("Our test description"),
         "goodsIdentification.typeOfPackages" -> Seq("boxes"),
         "goodsIdentification.numberOfPackages" -> Seq("13"),
-        "goodsIdentification.shippingMarks" -> Seq("crosses")
+        "goodsIdentification.shippingMarks" -> Seq("crosses"),
+        "goodsIdentification.combinedNomenclatureCode.id" -> Seq("180"),
```

Comment:
Is it worth varying this value in the test?

Reply:
Agreed. Will vary.